src/db/image_repository.py

```
@@ -1,18 +1,28 @@
-def upsert_image(conn, origin_id, origin_table, s3_key, s3_bucket, image_category):
+from image_event import image_event_contract_pb2
+
+def upsert_image(conn, protobuf_message):
     print("upserting image")
     cursor = conn.cursor()
-    postgres_insert_query = """ INSERT INTO li_images.images (origin_id, origin_table, s3_key, s3_bucket, image_category)
+    postgres_insert_query = sql_query
+    record_to_insert = (protobuf_message.upsertImageEvent.originId,
+                        protobuf_message.upsertImageEvent.originTable,
+                        protobuf_message.upsertImageEvent.s3Key,
+                        protobuf_message.upsertImageEvent.s3Bucket,
+                        image_event_contract_pb2.ImageEvent.ImageCategory.Name(protobuf_message.upsertImageEvent.imageCategory))
+
+    print("executing query: \n" + str(postgres_insert_query) + "\nwith values \n" + str(record_to_insert))
+
+    try:
+        cursor.execute(postgres_insert_query, record_to_insert)
+        conn.commit()
+    except Exception as ex:
```

I think we might want to let this bubble up to the handler?  I think in this case we definitely want lambda to know this insert failed to events can get put on a DLQ